src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -71,6 +71,7 @@ public static class EnvironmentSettingNames
         public const string HealthPingEnabled = "WEBSITE_FUNCTIONS_HEALTH_PING_ENABLED";
         public const string TestDataCapEnabled = "WEBSITE_FUNCTIONS_TESTDATA_CAP_ENABLED";
         public const string AzureMonitorCategories = "WEBSITE_FUNCTIONS_AZUREMONITOR_CATEGORIES";
+        public const string AzureMonitorTimeIsoFormatEnabled = "FUNCTIONS_AZUREMONITOR_TIME_ISOFORMAT_ENABLED";
```

Comment:
Couple issues with the env name here:

1. This only impacts linux, but the name does not reflect that
2. The class that consumes this appears more generic then "azure monitor". Is it accurate to include that naming here, or should the name also be more generic? Or does this class directly correlate to AzureMonitor and no other telemetry technology?

Reply:
Yes, we're using this only in the LogAzureMonitorDiagnosticLogEvent method which is used to generate azmon logs. Verified the kusto logs and those are not impacted by this changes.

Was just wondering if the long names can cause any size limits etc, but otherwise I can go with this as Pragna suggested: WEBSITE_FUNCTIONS_AZUREMONITOR_LINUX_APPSERVICE_TIME_ISOFORMAT_ENABLED

Reply:
Yeah, that name is quite long. Reading the issue more I think I am less worried about confusion: ISO format is already enabled on other skus/platforms, so having a customer set this to "true" for those other skus won't be concerning. I am not too concerned about a customer setting this to "false" and we still emit ISO.

This brings up another question: I assume we are only going with the config value here to avoid a breaking change? But ideally this would be the default format?